src/condor_tools/htcondor_cli/eventlog.py

```diff
@@ -68,40 +91,150 @@ def __init__(self, logger, log_file, **options):
+                else:
+                # print header
+                    print(f"{'Site':<20} {'Evictions':<12} {'Wall Time':<12} {'Good Time':<12} {'CPU Usage':<12} {'Job Starts':<12} {'Job Successes':<14} {'Job Failures':<12}")
+                    # print stats for each site
+                    for site, stats in site_stats.items():
+                        if site:
+                            print(f"{site:<20} {stats['Evictions']:<12} {stats['Wall Time']:<12} {stats['Good Time']:<12} {stats['CPU Usage']:<12} {stats['Starts']:<12} {stats.get('Successes', ''):<14} {stats.get('Failures', ''):<12}")
+        elif options.get("json", True) or options.get("csv", True):
+            # add event summaries as objects to list for JSON output
+            json_output = []
+            for job_id, event_summary in event_summaries.items():
+                # convert cpu usage, wall time, and good time to seconds
+                cpu_usage = event_summary.get("CPU Usage", "0+00:00:00")
+                cpu_usage_seconds = int(cpu_usage.split("+")[1].replace(":", ""))
+                wall_time = event_summary.get("Wall Time", "0+00:00:00")
+                wall_time_seconds = int(wall_time.split("+")[1].replace(":", ""))
+                good_time = event_summary.get("Good Time", "0+00:00:00")
+                good_time_seconds = int(good_time.split("+")[1].replace(":", ""))
+                # replace cpu usage, wall time, and good time with seconds
+                event_summary["CPU Usage"] = cpu_usage_seconds
+                event_summary["Wall Time"] = wall_time_seconds
+                event_summary["Good Time"] = good_time_seconds
+                json_output.append({"Job ID": job_id, **event_summary})
+            if options.get("csv", True):
+                print("Job ID,Host,Start Time,Evict Time,Evictions,Wall Time,Good Time,CPU Usage")
+                for job_id, event_summary in event_summaries.items():
+                    print(f"{job_id},{event_summary.get('Host', '')},{event_summary.get('Start Time', '')},{event_summary.get('Evict Time', '')},{event_summary.get('Evictions', '')},{event_summary.get('Wall Time', '')},{event_summary.get('Good Time', '')},{event_summary.get('CPU Usage', '')}")
+            else:
+                print(json.dumps(json_output, indent=4))
+        else:
+            # determine maximum length of each column
+            max_job_id_len = max(len(str(job_id)) for job_id in event_summaries.keys())+2
+            max_host_len = max(len(data.get("Host", "")) for data in event_summaries.values())+2
+            max_start_time_len = max(len(data.get("Start Time", "")) for data in event_summaries.values())+2
+            max_evict_time_len = max(len(data.get("Evict Time", "")) for data in event_summaries.values())+2
+            max_evictions_len = len(str("Evictions"))+2
+            max_time_format = len(str("0+00:00:00"))+2
```

Comment:
Does HTCondor actually truncate at one day?

Reply:
(You may need to fix this in the code you copied this code from, which is a big part of the reason to not copy code...)

Reply:
> Does HTCondor actually truncate at one day?

What do you mean by this? For most of the time columns like wall time  and good time, they are calculated by getting the time of the event and subtracting the start time.

Reply:
Sorry, I typed that wrong.  I should have said "truncate at one _digit_ of days?"  The computation of `max_time_format` doesn't allow for jobs that run for, say, two weeks (e.g., `14+00:00:01`).

I'm not really worried about not having enough space for jobs that run for more than 100 days.

Reply:
Ok, I think that all that would happen right now is the formatting would be different, but the time would still be correct. I will change the format to account for 2 days tho.

Reply:
> (You may need to fix this in the code you copied this code from, which is a big part of the reason to not copy code...)

Are you saying that this logic should just be extracted to a function?

Reply:
It now looks like you only have this code in one place, so you don't need to make it a function.  Not sure if one of your updates moved it, or if I just misread the initial diff; if so, sorry for the confusion.